app/models/prison.rb

```diff
@@ -87,8 +120,6 @@ def each
       end
     end
 
-  private
-
```

Comment:
STET - we shouldn't be removing private keywords

Reply:
Hi @starswan I removed it because there was already a private key word. Is it ok to have 2?